ext/azuredevops/setupAzd/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "setup-azd",
+  "version": "1.0.0",
```

Comment:
Should this really be 1.0.0?

Reply:
```suggestion
  "version": "0.12.0",
```

Reply:
It looks like the version here doesn't really matter to the extension or the task.
The extension version is set on the top level `vss-extension.json` file and the task version is defined on `task.json` file within the `setupAzd`. 
We can keep it in sync, but I think it would be only relevant in case we would want to push this to npm?